CHANGELOG.md

```diff
@@ -21,9 +63,39 @@
 
   Please update any configurations to use `quay.io/signalfx/splunk-otel-collector:<version>` for this and future releases.
 
-### 🧰 Bug fixes 🧰
+### 💡 Enhancements 💡
+
+- (Splunk) `discovery`: Update redis discovery instructions ([#4915](https://github.com/signalfx/splunk-otel-collector/pull/4915))
+- (Splunk) `discovery`: Bring Kafkamatrics receiver into the discovery mode ([#4903](https://github.com/signalfx/splunk-otel-collector/pull/4903))
+- (Contrib) `pkg/ottl`: Add the `Day` Converter to extract the int Day component from a time.Time ([#33106](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33106))
+- (Contrib) `pkg/ottl`: Adds `Month` converter to extract the int Month component from a time.Time (#33106) ([#33106](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33106))
+- (Contrib) `cmd/telemetrygen`: Add support for adding spanID and traceID as exemplars to datapoints generated by telemetrygen ([#33320](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33320))
+- (Contrib) `cmd/telemetrygen`: Add support for specifying trace ID and span ID in telemetrygen for logs ([#33234](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33234))
```

Comment:
Our only dependency is in some of our examples, I'm not sure whether or not these should be included. Maybe someone else has opinions here?

Reply:
@pjanotti what do you think?

Reply:
We don't ship the telemetrygen. No reason to mention it

Reply:
removing it.